src/Backend/__mocks__/WASMWrapper.js

```diff
@@ -1,5 +1,11 @@
 import {mockPrograms, mockMaterials, mockIons} from '../../__test__/MockData'
 
+export const StoppingPowerUnits = {
```

Comment:
This is already defined in `src/Backend/WASMWrapper.js` 
Why do we need another copy here ?

Reply:
This mocks the whole module and i'd argue it's never good to use the actual data in mock modules. If we need to change this just for tests this grants us a way to do it

Reply:
OK, thanks for explanation !